src/test/java/com/mesosphere/velocity/marathon/MarathonStepTest.java

```diff
@@ -289,14 +233,14 @@ private String generateSimpleScript() {
      * @param id           marathon id for application (optional)
      * @return pipeline groovy script
      */
-    private String generateSimpleScript(final String fileContents, final String id) {
+    private String generateSimpleScript(final String fileContents, final String id) throws IOException {
         final String nodeScript = "node { \n" +
                 "writeFile(encoding: 'utf-8', file: 'marathon.json', text: \"\"\"%s\"\"\");\n" +
                 "marathon(%s url: '%s');\n" +
```

Comment:
can you do named parameters?
can you do join w/ a comma instead of forcing a comma?